scalyr_agent/log_processing.py

```diff
@@ -27,6 +27,9 @@
 
 import sys
 
+if False:
+    from typing import Union, Optional
+
```

Comment:
Is this needed?

Reply:
Yes, lint complains about this import missing.